index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>New Tab Extension</title>
+        <script src="./script.js" type="text/javascript"></script>
+        <link rel='stylesheet' href="./style.css" type='text/css'/>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <h1>New Tab</h1>
+                <ul class="nav-links">
+                    <li><a href="https://www.amazon.com/">Amazon</a></li>
+                    <li><a href="https://www.ebay.com">Ebay</a></li>
+                    <li><a href="https://www.microsoft.com">Microsoft</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <div class="container">
```

Comment:
Hey, I see this <div> tag open without anything inside of it, it could've been left out.